control_panel_api/aws.py

```diff
@@ -24,20 +26,20 @@ def put_bucket_logging(name, target_bucket, target_prefix):
 
 
 def create_policy(name, policy_document):
-    boto3.client("iam").create_policy(
+    aws_api_client("iam").create_policy(
         PolicyName=name,
         PolicyDocument=json.dumps(policy_document)
     )
 
 
 def delete_policy(policy_arn):
-    boto3.client("iam").delete_policy(PolicyArn=policy_arn)
+    aws_api_client("iam").delete_policy(PolicyArn=policy_arn)
 
 
 def detach_policy_from_entities(policy_arn):
     """Get all entities to which policy is attached
-    See: http://boto3.readthedocs.io/en/latest/reference/services/iam.html#IAM.Client.list_entities_for_policy"""
-    entities = boto3.client("iam").list_entities_for_policy(PolicyArn=policy_arn)
+    See: http://boto3.readthedocs.io/en/latest/reference/services/iam.html#IAM.list_entities_for_policy"""
```

Comment:
This URL is was correct.  Not sure this doc comment is very useful anyway.

Reply:
removed